SAST/jenkins-mend_sast.yml

```diff
@@ -24,13 +24,11 @@ pipeline {
     }
                        
     stage('Run Mend SAST') {
-      steps {
-          sh '''
+      steps { 
         repo=$(basename -s .git $(git config --get remote.origin.url))
         branch=$(git branch --show-current)
-        ./mendsastcli --dir ./ --name=$repo_$branch --app=$repo
-        '''
-          }
+        sh './mendsastcli --dir ./ --name=$repo_$branch --app=$repo'
       }
     }
-  }
\ No newline at end of file
+  }
+} 
```

Comment:
Was this example tested?
Is the SAST_SERVER URL correct?
Is the syntax of the shell script in the 'Run Mend SAST' step correct and working?